problems/prob2.rb

```diff
@@ -0,0 +1,28 @@
+#Given an integer array of size N, identify the largest 2 numbers in this array.
+#Do not sort the array.
+
+def largest_two_numbers(numbers)
+  if numbers.empty?
+    0
```

Comment:
The return is not consistent, if you sometime return a single integer and sometime an array will confuse the caller a lot.